src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -28,13 +27,22 @@ public static void BasicAuthentication_RoundTrips_Echo()
         {
             BasicHttpBinding basicHttpBinding = new BasicHttpBinding(BasicHttpSecurityMode.Transport);
             basicHttpBinding.Security.Transport.ClientCredentialType = HttpClientCredentialType.Basic;
-            Action<ChannelFactory> credentials = (factory) =>
-            {
-                factory.Credentials.UserName.UserName = s_username;
-                factory.Credentials.UserName.Password = s_password;
-            };
 
-            ScenarioTestHelpers.RunBasicEchoTest(basicHttpBinding, Endpoints.Https_BasicAuth_Address, "BasicHttpBinding - Basic auth ", errorBuilder, credentials);
+            Logger.LogInformation("Starting basic echo test.\nTest variation:...\n{0}\nUsing address: '{1}'", "BasicAuthentication_RoundTrips_Echo", Endpoints.Https_BasicAuth_Address);
+            ChannelFactory<IWcfCustomUserNameService> factory = new ChannelFactory<IWcfCustomUserNameService>(basicHttpBinding, new EndpointAddress(Endpoints.Https_BasicAuth_Address));
+            factory.Credentials.UserName.UserName = "test1";
+            factory.Credentials.UserName.Password = "1tset";
+
+            IWcfCustomUserNameService serviceProxy = factory.CreateChannel();
+
+            string testString = "I am a test";
+            string result = serviceProxy.Echo(testString);
+            bool success = string.Equals(result, testString);
+
+            if (!success)
+            {
+                errorBuilder.AppendLine(String.Format("    Error: expected response from service: '{0}' Actual was: '{1}'", testString, result));
+            }
         }
         catch (Exception ex)
```

Comment:
Agree with @iamjasonp  -- we need a negative test that catches exactly the kind of exception we expect for a service that throws SecurityTokenException for an invalid user name or password.  A negative test that validates the ArgumentNullException path would also be good to include